### PR TITLE
Add support for Multi master

### PR DESCRIPTION
**Pull request by dineshba**
Add support to create **multi-master**

- User can provide the number of masters (default: 1)
- User should create the loadbalancer beforehand with proper proxy rules
- If count >1, then loadbalancer_address is mandatory
- If count ==1, then loadbalancer_address is optional. If provided, then it will use it

Screenshot 2020-10-24 at 9 41 37 PM

### CODE REVIEW

File: cluster/playbook/kubernetes.yaml [hunk 1]
```diff
@@ -13,21 +13,32 @@
...
+    - {role: kubernetes/master/primary}
     - {role: kubernetes/addons/cni}
     - {role: kubernetes/addons/ingress, when: addons.ingress.enabled }
     - {role: kubernetes/addons/csi, when: provider == "digitalocean" and addons.csi.enabled }
     - {role: kubernetes/addons/ccm, when: provider == "digitalocean" and addons.ccm.enabled }
     - {role: kubernetes/addons/external_dns, when: provider == "digitalocean" and addons.external_dns.enabled }
     - {role: kubernetes/addons/cert_manager, when: provider == "digitalocean" and addons.cert_manager.enabled }
 
+- name: Configure kubernetes secondary master(s)
+  hosts: master[1:]
```
Comment (dineshba) [inline cluster/playbook/kubernetes.yaml:33]: @jskswamy Thanks for the comments. I have adopted like below:

Using `master[1:]` to run the role master/common, master/secondary
Using `master[0]` to run the role master/common, master/primary

I have tested with this new roles

File: cluster.do.yaml [hunk 2]
```diff
@@ -15,16 +15,17 @@ addons:
 cluster:
   provider : digital_ocean
   master:
+    count: 2
     image: ubuntu-18-04-x64
     region: blr1
     size: s-2vcpu-4gb
   node_pools:
-    - count: 1
+    - count: 0
```
Comment (jskswamy) [inline cluster.do.yaml]: can we leave it to the default value of 1

File: cluster.do.yaml [hunk 3]
```diff
@@ -15,16 +15,17 @@ addons:
...
       image: ubuntu-18-04-x64
       region: blr1
       size: s-2vcpu-4gb
       name: medium
-    - count: 1
+    - count: 0
```
Comment (jskswamy) [inline cluster.do.yaml]: can we leave it to the default value of 1

File: cluster.do.yaml (hunk 2 above)
Comment (dineshba) [reply cluster.do.yaml]: Done

File: cluster.do.yaml (hunk 3 above)
Comment (dineshba) [reply cluster.do.yaml]: Done

**Comment by prabhu43** [comment]
LGTM

**Review by prabhu43 — COMMENTED** [review]
LGTM

**Comment by jskswamy** [comment]
Looks good to me as well @dineshba thanks a lot for this contribution

**Pull request merged**